src/main/java/com/todayhouse/domain/story/api/StoryReplyController.java

```diff
@@ -0,0 +1,43 @@
+package com.todayhouse.domain.story.api;
+
+import com.todayhouse.domain.story.application.StoryReplyService;
+import com.todayhouse.domain.story.dto.reqeust.CreateReplyRequest;
+import com.todayhouse.domain.story.dto.reqeust.DeleteReplyRequest;
+import com.todayhouse.domain.story.dto.response.CreateReplyResponse;
+import com.todayhouse.domain.story.dto.response.ReplyGetResponse;
+import com.todayhouse.domain.user.domain.User;
+import com.todayhouse.global.common.BaseResponse;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.security.core.annotation.AuthenticationPrincipal;
+import org.springframework.web.bind.annotation.*;
+
+@RequestMapping("/stories")
+@RestController
+@RequiredArgsConstructor
+public class StoryReplyController {
+    private final StoryReplyService replyService;
+
+    @PostMapping("/reply")
+    public BaseResponse<CreateReplyResponse> replyPost(@RequestBody CreateReplyRequest request, @AuthenticationPrincipal User user) {
+        CreateReplyResponse response = replyService.replyStory(user, request);
+        return new BaseResponse<>(response);
+
+    }
+
+    @DeleteMapping("/reply")
+    public BaseResponse<String> deleteReply(@AuthenticationPrincipal User user, @RequestBody DeleteReplyRequest request) {
+        replyService.deleteReply(user, request);
+        return new BaseResponse<>("삭제 완료");
+    }
+
+    @GetMapping("/reply")
+    public BaseResponse<Page<ReplyGetResponse>> findReplies(@RequestParam Long storyId, Pageable pageable) {
+        Page<ReplyGetResponse> replies = replyService.findReplies(storyId, pageable);
+        return new BaseResponse<>(replies);
+    }
```

Comment:
```suggestion
    public BaseResponse<PageDto<ReplyGetResponse>> findReplies(@RequestParam Long storyId, Pageable pageable) {
        Page<ReplyGetResponse> replies = replyService.findReplies(storyId, pageable);
        PageDto<ReplyGetResponse> responses = new PageDto<>(replies);
        return new BaseResponse<>(responses);
    }
```
common package의 PageDto를 사용하면 필요한 정보만 보낼 수 있습니다.